lib/job/arObjectMoveJob.class.php

```diff
@@ -120,4 +122,13 @@ public function runJob($parameters)
 
         return true;
     }
+
+    public function reindexObjectRange(array $children, int $start, int $end)
+    {
+        for ($i = $start; $i <= $end; ++$i) {
+            $objectId = $children[$i]['id'];
+            $this->info(sprintf('Reindexing object id: %d', $objectId));
+            QubitSearch::getInstance()->update(QubitObject::getById($objectId));
```

Comment:
If `QubitObject::getById($objectId)` returns `null` because the row has been deleted from the database, this will throw an error.

Reply:
👍